addons/hr_expense/security/ir_rule.xml

```diff
@@ -2,21 +2,62 @@
 <odoo>
     <data noupdate="1">
 
+        /* Expense rules */
```

Comment:
Instead of adding (JS-like) comments, I would rename xml_ids so that it will be so much clear to everyone
ir_rule_hr_expense_manager
ir_rule_hr_expense_user
ir_rule_hr_expense_sheet_manager
ir_rule_hr_expense_sheet_user
ir_rule_hr_expense_sheet_employee